libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -143,7 +143,6 @@ export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvide
     // These keys are public to permit access to the keys from the adapter when it's a being
     // from library that does not have access to static properties off of BotFrameworkAdapter.
     // E.g. botbuilder-dialogs
-    public readonly BotIdentityKey: Symbol = Symbol('BotIdentity');
```

Comment:
Ugh who wrote this code? :tired_face:

I've been looking at the C# BotFrameworkAdapter this week and saw that I put this key in the wrong location, thanks for fixing this!